src/js/coblocks-lightbox.js

```diff
@@ -116,8 +116,8 @@
 						imagePreloader[ `img-${ imgIndex }` ] = new window.Image();
 						imagePreloader[ `img-${ imgIndex }` ].src = img.attributes.src.value;
 						imagePreloader[ `img-${ imgIndex }` ][ 'data-caption' ] =
-							( images[ imgIndex ] && images[ imgIndex ].nextElementSibling )
-								? getImageCaption( images[ imgIndex ] ) : '';
+								( images[ imgIndex ] && images[ imgIndex ].nextElementSibling )
```

Comment:
why are we changing the spacing here? It looks like the previous version (one space) was good

Reply:
Hey Jason. This one was actually made by the ESLint rules that I have loaded within my editor Visual Studio Code. We use the same rules that are defined from Core with a few minor modifications, so occasionally when those rules change, we see code shifts like this. 

https://github.com/godaddy-wordpress/coblocks/blob/master/.eslintrc.js